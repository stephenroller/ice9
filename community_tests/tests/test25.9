# if - true -> write 1; [] - false -> write 2; fi write 3;

if - true -> 
	write 1; 
[] - false -> 
	write 2; 
fi

write 3;

if - false -> 
	write 1; 
[] - true -> 
	write 2; 
fi

write 3;
