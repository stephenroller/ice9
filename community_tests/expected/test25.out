Loading file: generated/test25.tm
2 
3 
1 
3 
Number of instructions executed = 30
